src/bandersnatch/package.py

```diff
@@ -133,68 +141,71 @@ def save_json_metadata(self, package_info: Dict) -> bool:
 
         return True
 
-    async def sync(self, stop_on_error: bool = False, attempts: int = 3) -> None:
-        loop = asyncio.get_event_loop()
-        self.tries = 0
+    async def fetch_metadata(self, attempts: int = 3) -> Any:
```

Comment:
So the reason this type annotation is overly-broad is because ```master.get_package_metadata()``` returns ```Any```, and it does so because ```master.get()``` returns ```Any```, and I'm not sure if we can change the return value there.

But this is also my first experience with mypy and type annotations in Python so let me know if there is anything we can do to get around that.